MangoPay/PayOut.php

```diff
@@ -19,12 +33,24 @@ class PayOut extends Transaction
      */
     public $PaymentType;
 
+	/**
+	 * One of PayOutPaymentDetails implementations, depending on $PaymentType
+	 * @var PayOutPaymentDetailsBankWire
+	 */
+	public $PayoutPaymentDetails;
+
```

Comment:
did you manage to setup a rule in the linter to avoid such bad indents ?

Reply:
no, Would you help me?